pages/tools/argent-x.mdx

```diff
@@ -0,0 +1,62 @@
+import { Steps } from 'nextra-theme-docs'
+
+# Argent X Overview
+[Argent X](https://www.argent.xyz/argent-x/) is an open-source Starknet wallet utilizing account abstraction for a secure and user-friendly experience. It allows NFT management, interaction with staking and DeFi protocols, and direct crypto purchases within the wallet. 
+
+## Installing Argent X 
+Follow the official Argent X installation instructions:<br></br>
+`https://www.argent.xyz/learn/how-to-create-an-argent-x-wallet/`
+
+## Use Argent X with Madara
+Argent X includes the Mainnet, Sepolia, and Goerli networks by default, but connecting with your local Madara chain requires manual configuration. This involves adding a custom network within Argent X's settings.
+
+### Configuring Argent X for Madara
+<Steps>
+### Open the Argent X wallet and navigate to Settings.
+![stack](../../static/argentX/step1.png)
+### Select "Developer settings" and then "Manage networks".
+![stack](../../static/argentX/step2.png)
+### Click the plus button on the top right to add a network.
+![stack](../../static/argentX/step3.png)
+### Fill in the following fields:
+    - Network Name: A friendly name for the Madara network.
+    - Chain ID: The default chain ID on Madara is `SN_GOERLI`, to add a custom chain ID, refer to the Configure section of Madara documentation<br></br>
+        If you want to verify the chain ID, follow these steps:<br></br>
```

Comment:
the chain id subsection doesn't look well formatted like this. i guess let's just move it its own page where we can show - fetching the chainId & changing the chainId